src/components/_pages/certificates/detail/index.tsx

```diff
@@ -957,6 +960,69 @@ export default function CertificateDetail() {
         return sanList;
     }, [certificate]);
 
+    const csrPropertiesData: TableDataRow[] = useMemo(() => {
+        return certificate?.certificateRequest
+            ? [
+                  {
+                      id: "commonName",
+                      columns: ["Common Name", certificate?.certificateRequest?.commonName || ""],
+                  },
+                  {
+                      id: "certificateType",
+                      columns: [
+                          "Certificate Type",
+                          certificate?.certificateRequest?.certificateType
+                              ? getEnumLabel(contentTypeEnum, certificate?.certificateRequest?.certificateType)
```

Comment:
you are searching for labels with same enum (contentTypeEnum) for two different enum types (certificateType and certificateRequestFormat on line 983), that seems wrong